Zh translation, almost all messages.

wqmeng opened the pull request:
Please merge. 
Thank you.

wqmeng committed:
Update emulationstation2.po

wqmeng committed:
Update emulationstation2.po

wqmeng committed:
Update emulationstation2.po

shantigilbert commented:
duplicate #61 ?

wqmeng commented:
Hello

My post including all the msgids in @lihexali translation.  I do not see his pull request when I start to translate.  You can close that one and only merge mine.  If @lihexali think that some of my translations are not very suitable, any correct is very welcome.

Thank you.

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -92,10 +92,10 @@ msgid "enter path to map"
 
 msgid "Bezel (16:9)"
-msgstr ""
+msgstr "遮罩（16:9）"
```
Bezel 叫做 边框 应该是更合适的吧，下面的msgid "enter path to bezel (16:9)"也是

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -92,10 +92,10 @@ msgid "enter path to map"
 
 msgid "enter path to bezel (16:9)"
-msgstr ""
+msgstr "输入遮罩（16:9）路径"
```
同样的，Bezel更适合被翻译作 边框

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -324,16 +324,16 @@ msgid "BLUETOOTH SCAN IS ALREADY RUNNING."
 
 msgid "Formatting"
-msgstr ""
+msgstr "格式化"
```
正在格式化

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -324,16 +324,16 @@ msgid "BLUETOOTH SCAN IS ALREADY RUNNING."
 
 msgid "FORMATTING DEVICE"
-msgstr ""
+msgstr "格式化设备"
```
正在格式化设备

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -324,16 +324,16 @@ msgid "BLUETOOTH SCAN IS ALREADY RUNNING."
 
 msgid "A DRIVE IS ALREADY BEING FORMATTED."
-msgstr ""
+msgstr "驱动器已被格式化。"
```
驱动器已在被格式化

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -372,51 +372,53 @@ msgid "HARDCORE MODE"
 "Shows icons in the bottom right corner when eligible achievements can be "
 "earned."
 msgstr ""
+"在右下角显示图标当符合条件的成就可"
```
语序是否应该遵循中文习惯？
当可以获得符合条件的成就时，在右下角显示图标

这里表征的是比如要求你击杀三个敌人，当你击杀1个或者两个时，显示图标和1/3这样的进度

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -585,16 +587,16 @@ msgstr "请稍后"
 #, c-format
 msgid "%d%% (%d of %d)"
-msgstr ""
+msgstr "%d%% (%d 在 %d)"
```
这个形式应该是百分比，当三个%d参数分别以60, 3, 5这样的形式传入，字符串是 ``60% (3 of 5)``，更符合中文习惯的说法是 60% (3/5)，形式是 ``%d%% (%d/%d)``，60% (3 在 5) 看起来太怪了

shantigilbert commented:
You need to resolve the conflicts before I can merge

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1023,13 +1025,13 @@ msgstr "确定吗？"
 #, c-format
 msgid "UPDATING %s"
-msgstr ""
+msgstr "更新 %s"
```
whatever作为%s传入时，字符串是 ``UPDATEING whatever``, 符合中文习惯的说法是``更新 whatever 中``，或者``正在更新 whatever``。这里应该以``更新 %s 中``，或者``正在更新 %s``更合适

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1062,58 +1066,58 @@ msgid "INVALID PARAMETERS"
 
 msgid "HOST A NETPLAY GAME"
-msgstr ""
+msgstr "开服联网游戏"
```
主持联网游戏 是更适合的说法，或者开启联网游戏服务器，开服作为一个完整的动宾短语后面再跟名词很奇怪，不过也可以倒装到联网游戏开服，也许会更自然一些
对应替换概念，开启Minecraft服务器，正常的说法；主持Minecraft(多人游戏)，怪一点；开服Minecraft，非常怪；Minecraft开服，不再那么怪

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1062,58 +1066,58 @@ msgid "INVALID PARAMETERS"
 
 msgid "PUBLICLY ANNOUNCE GAME"
-msgstr ""
+msgstr "公开地宣告游戏"
```
公开通告游戏，或者广播游戏

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1123,74 +1127,74 @@ msgstr ""
 
 msgid "Unlocked on"
-msgstr ""
+msgstr "解锁时间"
```
这里有明显的介词on，更合适的说法是 解锁于；因为这个msgid不一定仅用于后接时间，也可能用于后接游戏名，用 解锁于 的话，两者都能适用

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1123,74 +1127,74 @@ msgstr ""
 
 msgid "SLOT"
-msgstr ""
+msgstr "档位"
```
存档的位置，叫做槽位更合适

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1223,94 +1227,94 @@ msgid "LOGO SOURCE"
 
 msgid "BEZEL (16:9)"
-msgstr ""
+msgstr "遮罩（16:9）"
```
依然，Bezel应该遵循习惯翻译作边框

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1223,94 +1227,94 @@ msgid "LOGO SOURCE"
 
 msgid "MAP"
-msgstr ""
+msgstr "地图"
```
这里是按键映射，不是地图

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1223,94 +1227,94 @@ msgid "LOGO SOURCE"
 
 msgid "LAST 15 DAYS"
-msgstr ""
+msgstr "前 15 天"
```
15天内

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1223,94 +1227,94 @@ msgid "LOGO SOURCE"
 
 msgid "LAST 3 MONTHS"
-msgstr ""
+msgstr "前 3 个月"
```
3个月内

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1223,94 +1227,94 @@ msgid "LOGO SOURCE"
 
 msgid "SCRAPE NOW"
-msgstr "开始抓取"
+msgstr "开始刮削"
```
含有now的短语，立即刮削应当更合适

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1343,13 +1350,13 @@ msgid "SYSTEM"
 
 msgid "THE BEZEL PROJECT"
-msgstr "框架项目"
+msgstr "装饰框项目"
```
这里的The Bezel Project是一个具体的项目名，https://github.com/thebezelproject
可以不翻译，或者一定要翻译的话也应该是边框项目

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1343,13 +1350,13 @@ msgid "SYSTEM"
 
 msgid "SELECT BEZELS TO INSTALL/REMOVE"
-msgstr ""
+msgstr "选择要安装/移除的遮罩"
```
选择要安装/移除的边框

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1516,7 +1523,7 @@ msgid "SHOW FRAMERATE"
 
 msgid "Also turns on the emulator's native FPS counter, if available."
-msgstr ""
+msgstr "如果可用，将会打开模拟器的本机 FPS 计数器。"
```
native 原生
如果可用，也开启模拟器的原生FPS计数器(机翻)/显示(意译)

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1547,13 +1554,13 @@ msgid "CREATE A SUPPORT FILE"
 
 msgid "CREATE A SUPPORT FILE? THIS INCLUDES ALL DATA IN YOUR SYSTEM FOLDER."
-msgstr ""
+msgstr "创建支持文件？ 这包括您游戏系统文件夹中的所有数据。"
```
询问句应以 是否 开头
是否创建支持文件？这包括你系统文件夹中的所有数据
不是游戏系统文件夹(/storage/roms/nes)，而是系统文件夹(/emuelec)

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1580,29 +1587,30 @@ msgid "REDETECT ALL GAMES' LANG/REGION"
 
 msgid "IGNORE MULTI-FILE DISK CONTENT (CUE/GDI/CCD/M3U)"
-msgstr ""
+msgstr "忽略多文件磁盘内容 (CUE/GDI/CCD/M3U)"
```
游戏媒介以光盘为主(多盘PS, CDI, DC游戏)，但也有磁盘(多盘Commodore游戏)
也许多文件碟片内容更合适

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1622,33 +1632,35 @@ msgid "QUICK SYSTEM SELECT"
 
 msgid "SWITCH A & B BUTTONS IN EMULATIONSTATION"
-msgstr "在EMULATIONSTATION中切换 A/B 按键"
+msgstr "在 EMULATIONSTATION 中切换 A/B 按键"
```
交换AB键

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1705,55 +1720,58 @@ msgid "START UPDATE"
 
 msgid "SHOW CLOCK IN 12-HOUR FORMAT"
-msgstr ""
+msgstr "以 12 小时制显示时钟"
```
12小时制 是一个完整的名词，这里不应该有明显的空格，而且以中文习惯不应该在阿拉伯数字左右留空，直接以12小时制显示时钟就好

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1780,40 +1798,40 @@ msgid "AUDIO OUTPUT"
 
 msgid "BOOT SPLASH"
-msgstr ""
+msgstr "启动闪屏"
```
机翻，开机屏幕；意译，开机视频/开机画面
这里的概念是bootsplash/bootscreen: https://en.wikipedia.org/wiki/Bootsplash

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1970,118 +1990,118 @@ msgid "DECORATIONS"
 
 msgid "NORTH WEST"
-msgstr ""
+msgstr "北西"
```
这里和下面，方向应遵循中文习惯，西北，东北，等等

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1970,118 +1990,118 @@ msgid "DECORATIONS"
 
 msgid "ENABLE AI TRANSLATION SERVICE"
-msgstr ""
+msgstr "启用人工智能翻译服务"
```
要么统一使用AI，要么统一使用人工智能，不要部分使用AI部分使用人工智能

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2249,16 +2274,16 @@ msgid "SHOW BATTERY STATUS"
 
 msgid "Ignore 'The' and 'A' if at the start."
-msgstr ""
+msgstr "忽略 'The' 和 'A' 如果在开头。"
```
忽略开头的The和A

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2336,16 +2361,16 @@ msgid "QUICK ACCESS"
 
 msgid "NOW PLAYING"
-msgstr ""
+msgstr "现在播放"
```
正在播放

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2384,16 +2409,16 @@ msgid "AUTO SAVE/LOAD ON GAME LAUNCH"
 
 msgid "SHOW CONTROLLER OVERLAYS"
-msgstr ""
+msgstr "显示控制器遮罩"
```
显示控制器叠加层
这个是显示在屏幕一角的小手柄标志

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2384,16 +2409,16 @@ msgid "AUTO SAVE/LOAD ON GAME LAUNCH"
 
 msgid "OVERLAY CORNER"
-msgstr ""
+msgstr "遮罩角"
```
叠加层角落
这个是显示在屏幕一角的小手柄标志，选择4个不同角落

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2417,7 +2442,7 @@ msgid "INTERNAL RESOLUTION"
 
 msgid "LAUNCH THIS GAME AT STARTUP"
-msgstr ""
+msgstr "在启动时启动此游戏"
```
如果两个启动表示不同意思，最好用不同的动词
在开机时启动该游戏

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2476,22 +2503,22 @@ msgid "ERROR: EVERY SYSTEM IS HIDDEN, RE-DISPLAYING ALL OF THEM NOW"
 
 msgid "GO TO LETTER"
-msgstr ""
+msgstr "按字母"
```
跳转到字母

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2476,22 +2503,22 @@ msgid "ERROR: EVERY SYSTEM IS HIDDEN, RE-DISPLAYING ALL OF THEM NOW"
 
 msgid "GO TO MANUFACTURER"
-msgstr "访问制造商"
+msgstr "按制造商"
```
跳转到厂商

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2476,22 +2503,22 @@ msgid "ERROR: EVERY SYSTEM IS HIDDEN, RE-DISPLAYING ALL OF THEM NOW"
 
 msgid "GO TO HARDWARE"
-msgstr ""
+msgstr "按硬件"
```
跳转到硬件

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2476,22 +2503,22 @@ msgid "ERROR: EVERY SYSTEM IS HIDDEN, RE-DISPLAYING ALL OF THEM NOW"
 
 msgid "GO TO DECADE"
-msgstr ""
+msgstr "按年代"
```
跳转到年代

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2522,27 +2549,33 @@ msgid ""
 "FORMAT.\n"
 "DO YOU WANT TO LAUNCH IT ANYWAY?"
 msgstr ""
+"警告：当前设置的模拟器/核心不支持此游戏的文件"
```
这里，以及前前后后，msgstr支持多行文本，翻译后应以翻译文本实际内容换行

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2522,27 +2549,33 @@ msgid ""
 
 msgid "Full"
-msgstr "已满"
+msgstr "全部"
```
完整
这里和下面的几个是功能模式，完整，儿童，基本，这个样子

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2562,27 +2595,29 @@ msgstr "游玩次数"
 "A valid account is required. Please register an account on https://"
 "retroachievements.org"
 msgstr ""
+"需要一个有效的帐户。请注册一个帐户在 https://"
```
需要一个有效的账户，请在https://retroarchievements.org注册一个账户

以上，以实际内容宽度换行

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -2708,55 +2743,55 @@ msgid "Extracting"
 
 msgid "FAMILY"
-msgstr ""
+msgstr "家族"
```
派别/流派
类型->派别->具体作品

7Ji commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -3503,10 +3554,10 @@ msgid "START RETROARCH"
 
 msgid "REBOOT FROM NAND"
-msgstr "重启到 NAND"
+msgstr "从 NAND 重启（看电视）"
```
从NAND重启一定是看电视吗？NAND上面的系统一定是Android TV吗？这里没有必要额外补充细节，用户界面上的东西不要有类似于编者注这样的内容

7Ji reviewed (changes requested):
I think these strings need different translation or formatting

lihexali commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1223,94 +1227,94 @@ msgid "LOGO SOURCE"
 
 msgid "SCRAPER"
-msgstr "抓取元数据"
+msgstr "刮削器"
```
是个名词，翻译为"抓取器"比较合适

lihexali commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1223,94 +1227,94 @@ msgid "LOGO SOURCE"
 
 msgid "SCRAPING DATABASE"
-msgstr ""
+msgstr "刮削数据库"
```
数据库抓取中

lihexali commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1223,94 +1227,94 @@ msgid "LOGO SOURCE"
 
 msgid "SCRAPE FROM"
-msgstr "游戏数据来源"
+msgstr "刮削从"
```
抓取于

lihexali commented on locale/lang/zh_CN/LC_MESSAGES/emulationstation2.po:
```diff
@@ -1223,94 +1227,94 @@ msgid "LOGO SOURCE"
 
 msgid "GAMES TO SCRAPE FOR"
-msgstr ""
+msgstr "刮削的游戏"
```
为哪些游戏抓取

wqmeng committed:
Update emulationstation2.po

Base on English emulationstation2.pot file, update all message strings, correct a few translations by suggests. Fix merge conflict.

wqmeng committed:
Update emulationstation2.po
